src/api/store/community.py

```diff
@@ -183,29 +182,31 @@ def create_community(self,context: Context, args) -> (dict, MassEnergizeAPIError
           comm_admin.save()
 
       #also clone all global actions for this community
-      global_actions = Action.objects.filter(is_global=True)
-      for action_to_copy in global_actions:
-        action_to_copy_id = action_to_copy.id
-        old_tags = action_to_copy.tags.all()
-        old_vendors = action_to_copy.vendors.all()
-        new_action: Action = action_to_copy
-        new_action.pk = None
-        new_action.is_published = False
-        new_action.is_global = False
+      #global_actions = Action.objects.filter(is_global=True)
+      #for action_to_copy in global_actions:
+      #  print(action_to_copy)
+      #  action_to_copy_id = action_to_copy.id
+      #  old_tags = action_to_copy.tags.all()
+      #  old_vendors = action_to_copy.vendors.all()
+      #  new_action: Action = action_to_copy
+      #  new_action.pk = None
+      #  new_action.is_published = False
+      #  new_action.is_global = False
 
         #make sure the action title does not exceed the max length expected
-        suffix = f'-{new_community.subdomain}-{action_to_copy_id}'
-        new_action.title = new_action.title[:SHORT_STR_LEN-len(suffix)]+suffix 
+      #  suffix = f'-{new_community.subdomain}-{action_to_copy_id}'
```

Comment:
again why are you removing?  i am expecting at least a comment

Reply:
Now that we don't need to have unique action names, there is no reason to add the action_to_copy_id.  It is just a pain for community admins and takes away from the useful length of the name.